src/download/linux.md

```diff
@@ -224,7 +224,7 @@ RPM packages are available on the [Open Build Service](https://download.opensuse
   
 ```bash
 # add repository (if on leap, replace 'openSUSE_Tumbleweed' with '15.4')
-zypper addrepo https://download.opensuse.org/repositories/home:getchoo/openSUSE_Tumbleweed/home:getchoo.repo
```

Comment:
the current instructions are from obs directly (see [here](https://software.opensuse.org//download.html?project=home%3Agetchoo&package=prismlauncher)), so the url shouldn't need to be changed afaik. an alias could also be nice, but i'm not sure if it's exactly the best choice here as technically my obs home repo is being added and not one specific to prism. if i were to ever move it over though, i'd be up for this

Reply:
Must have been some weird copy-paste error on my end.. I could've sworn `zypper` complained when I tried adding the repo when the URL had the .repo at the end.

Reply:
it will if you include the repo name iirc, but otherwise it's fine